corehq/apps/reports/templates/reports/form/partials/single_form_tree.html

```diff
@@ -7,6 +7,12 @@
                     <strong>{% trans "Hidden Values (Form Calculations)" %}</strong>
                 </td>
             </tr>
+        {% elif 'commcare_usercase/' in question.value %}
+            <tr class="info">
+                <td colspan="2">
+                    <strong>{% trans "User Case Data" %}</strong>
+                </td>
+            </tr>
```

Comment:
where does the normal case data go? it seems like we should be doing the same thing as we do there but didn't see any references to it in this file.